tests/requirements.txt

```diff
@@ -0,0 +1 @@
+prospector==1.7.7
```

Comment:
Also, I don't think we need this right? The CI script manually installs it, anyway.

Reply:
Yes, it is installed in the gh action env